src/qibocal/protocols/characterization/ramsey.py

```diff
@@ -70,20 +74,18 @@ class RamseyData(Data):
     """Final delay between RX(pi/2) pulses in ns."""
     detuning_sign: int
     """Sign for induced detuning."""
-    nboot: int
-    """Number of bootstrap samples"""
     qubit_freqs: dict[QubitId, float] = field(default_factory=dict)
     """Qubit freqs for each qubit."""
     data: dict[QubitId, npt.NDArray] = field(default_factory=dict)
     """Raw data acquired."""
 
-    def register_qubit(self, qubit, wait, msr, phase):
+    def register_qubit(self, qubit, wait, prob, errors):
         """Store output for single qubit."""
         # to be able to handle the non-sweeper case
-        ar = np.empty(msr.shape, dtype=RamseyType)
+        ar = np.empty(np.shape(prob), dtype=RamseyType)
```

Comment:
```suggestion
        shape = (1,) if np.isscalar(prob) else prob.shape
        ar = np.empty(shape, dtype=RamseyType)
```
If you do this than you can remove also the cast to `np.array` in the acquisition (see comments below).